live/app/components/qroc-solution-panel.js

```diff
@@ -5,6 +5,16 @@ const QrocSolutionPanel = Ember.Component.extend({
   answer: null,
   solution: null,
 
+  isResultOk : Ember.computed('answer', function () {
```

Comment:
Je te propose le refacto suivant : 
  const classMap = {
    isResultOk: 'correction-qroc-box__input-right-answer',
    isResultKo: 'correction-qroc-box__input-wrong-answer',
    isResultWithoutAnswer: 'correction-qroc-box__input-no-answer'
  }

  resultInputClass : Ember.computed('answer', function () {
    return classMap[(this.get('answer.result'))] || '';
  })